### PR TITLE
Add spatial filter actions in Redux

### PR DESCRIPTION
**Pull request by Clebal**
**MERGE #247 BEFORE THIS ONE**

### CODE REVIEW

File: packages/react-redux/src/slices/cartoSlice.js [hunk 1]
```diff
@@ -93,6 +94,30 @@ export const createCartoSlice = (initialState) => {
...
+          const source = state.dataSources[sourceId];
+
+          if (source) {
+            source.spatialFilter = null;
+          }
+        } else {
+          state.spatialFilter = null;
+        }
+      },
       addFilter: (state, action) => {
```
Comment (padawannn) [inline packages/react-redux/src/slices/cartoSlice.js:121]: I think that `clearFilters` action should also remove the spatial filter for the datasource. Do you agree @VictorVelarde and @Clebal ?

**Review by padawannn — CHANGES_REQUESTED** [review]
I'm wondering if

File: packages/react-redux/src/slices/cartoSlice.js (hunk 1 above)
Comment (borja-munoz) [reply packages/react-redux/src/slices/cartoSlice.js:121]: I've not been summoned but I'd say it shouldn't. The `clearFilters` action should remove the filters added by the `addFilter` action. As we have decided to not treat the spatial filter like the other filters, we shouldn't remove it using this action.

Comment (Clebal) [reply packages/react-redux/src/slices/cartoSlice.js:121]: Agree with @borja-munoz

**Pull request closed without merging**